Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.xaml

```diff
@@ -133,7 +130,10 @@
                             FontStyle="{TemplateBinding FontStyle}"
                             FontWeight="{TemplateBinding FontWeight}"
                             Foreground="{TemplateBinding Foreground}"
-                            IsTabStop="False">
+                            IsTabStop="False"
+                            Height="{TemplateBinding Height}"
```

Comment:
Can we please remove this? I don't think we need this.

Reply:
I am referring to `Height="{TemplateBinding Height}"`

Reply:
Removing that binding causes the PointerOver size to be incorrect:

![image](https://user-images.githubusercontent.com/8283956/47608536-11405600-d9e4-11e8-8ac6-b007504de1de.png)

Here is what it looks like with that binding in place:

![image](https://user-images.githubusercontent.com/8283956/47608575-90358e80-d9e4-11e8-9c10-86d7a8aae65a.png)

I'm going to decline this change and resolve.

Reply:
We should almost never use `Height="{TemplateBinding Height}"` or `Height="{TemplateBinding ActualHeight}"` because they don't always work. In your case it only works because the height is already set to `40`.

The real reason that it's not stretched is because by default, the inner `Button` has `VerticalAlignment` set to `Center`. All you need to do is, replace the height binding with `VerticalAlignment="{TemplateBinding VerticalContentAlignment}"` and you can then set `<Setter Property="VerticalContentAlignment" Value="Stretch" />` on your `MenuItem` style. This way, the inner button automatically fills all the available space from the templated parent.

Reply:
Addressed